### ISSUE TITLE
Do we need an Task ID ?

### ISSUE THREAD

**Issue by aalbino2** [FAIRmat-NFDI/AreaA-data_modeling_and_schemas#11]
in this case the entry_id can't be enough to retrieve our task when the task is not a standalone entry, but rather an element of the tasks list.

**Comment by aalbino2** [comment]
We have now our Activity inheriting from ElnBaseSection, this means that the "lab_id" will end up also in each activity one produces.

Is this the meant behaviour? 
@hampusnasstrom @Pepe-Marquez

**Comment by hampusnasstrom** [comment]
I think this is how we intended it so that we also have identifiers for the activities that make sense. But again `lab_id` is a confusing name I guess. I overloaded it in my examples and gave it a better label like "Activity ID".

**Comment by aalbino2** [comment]
Yes I also found that lab_id for everything is confusing. Let’s discuss in some in-person meeting and let’s specify more than one ID!
